src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.js

```diff
@@ -321,9 +321,14 @@ const ImageClassificationObservationTable = ({ uploadedFiles, handleRemoveFile }
                     {subIndex === 0 && (
                       <>
                         <StyledTd rowSpan={numSubRows}>{imageIndex + 1}</StyledTd>
-                        <StyledTd rowSpan={numSubRows}>
+                        <TdWithHoverText
```

Comment:
accidentally removed the TdWithHoverText in a table re-design update. Now you can click on the image thumbnail and open the review modal when it's ready for review